### ISSUE TITLE
GPS integration

### ISSUE THREAD

**Issue by lukebrody** [kavigupta/urbanstats#466]
Use phone gps to show regions where you are

**Comment by lukebrody** [comment]
Quadtree broken into multiple files
Region rectangles in quad tree
More precise filtering on client

**Comment by kavigupta** [comment]
I think the data structure we want is as follows

Let R_n be a set of rectangles defined as

R_0 = whole world
R_n = every rectangle in R_{n-1} split into 4 smaller ones

Let R be the union over all R_n 

Assign a given shape s to the smallest rectangle

**Comment by kavigupta** [comment]
Hmmm ok same as above but you can split each element into 4 at a corner exactly once. Now an element will never be placed in a rectangle that's more than 2x the size of its bounding box

**Comment by kavigupta** [comment]
Yeah, I think this is good enough, you end up with 4x the minimal number of bytes used and 4/square compactness the minimal number of shapefile queries
